src/lib/operators/table_scan/base_single_column_table_scan_impl.hpp

```diff
@@ -25,12 +25,12 @@ class AttributeVectorIterable;
  */
 class BaseSingleColumnTableScanImpl : public BaseTableScanImpl, public AbstractColumnVisitor {
  public:
-  BaseSingleColumnTableScanImpl(const std::shared_ptr<const Table>& in_table, const ColumnID left_column_id,
+  BaseSingleColumnTableScanImpl(const std::shared_ptr<const Table>& in_table, const ColumnID column_id,
                                 const PredicateCondition predicate_condition);
 
   std::shared_ptr<PosList> scan_chunk(ChunkID chunk_id) override;
 
-  void handle_column(const ReferenceColumn& left_column, std::shared_ptr<ColumnVisitorContext> base_context) override;
+  void handle_column(const ReferenceColumn& base_column, std::shared_ptr<ColumnVisitorContext> base_context) override;
```

Comment:
it's not a BaseColumn, so this name might be confusing. Just "column" or leave it at "left_column"? (Does clang-tidy actually complain about this?)

Reply:
Mismatching parameter names (see above). Will change.